test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -62,14 +65,70 @@ void test_ucp_tag_xfer::test_xfer(xfer_func_t func, bool expected, bool sync)
     }
 }
 
+void test_ucp_tag_xfer::test_run_xfer(bool send_contig, bool recv_contig,
+                                      bool expected, bool sync)
+{
+    static const size_t count = 1148544;
```

Comment:
need to divide the size by ucs::test_time_multiplier() or skip with valgrind because it would take too long